plugins/inputs/system/disk.go

```diff
@@ -166,14 +167,13 @@ func (s *DiskIOStats) Gather(acc telegraf.Accumulator) error {
 var varRegex = regexp.MustCompile(`\$(?:\w+|\{\w+\})`)
 
 func (s *DiskIOStats) diskName(devName string) string {
+	// if len(s.NameTemplates) == 0 {
```

Comment:
Commented out so that the diskInfo code will still be exercised as before, will skip it when not using this option.